osdk/tools/build_image.sh

```diff
@@ -8,8 +8,15 @@ SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 ASTER_ROOT_DIR=${SCRIPT_DIR}/../..
 ASTER_RUST_VERSION=$( grep -m1 -o 'nightly-[0-9]\+-[0-9]\+-[0-9]\+' ${ASTER_ROOT_DIR}/rust-toolchain.toml )
 VERSION=$( cat ${ASTER_ROOT_DIR}/VERSION )
-IMAGE_NAME=asterinas/osdk:${VERSION}
-DOCKERFILE=${SCRIPT_DIR}/Dockerfile.ubuntu22.04
+DOCKERFILE=${SCRIPT_DIR}/ubuntu/Dockerfile
+
+if [ "$1" = "intel-tdx" ]; then
+    IMAGE_NAME="asterinas/osdk:${VERSION}-tdx"
+    ./gen_dockerfile.sh --intel-tdx --out-dir ubuntu
```

Comment:
Both TDX and non-TDX versions of the Dockerfile are under the `ubuntu` directory, rendering the directory useless. Just generate the Dockerfile in the current directory. Remember to add a `.gitignore` to prevent the generated files from being version controlled.